examples/container-deny-added-caps/src.rego

```diff
@@ -11,11 +11,13 @@ import data.lib.core
 import data.lib.pods
 import data.lib.security
 
+policyID := "P1001"
```

Comment:
thought: Probably doesn't matter a ton, but typically variables in rego use underscore casing if we want to be consistent with other policies.

Reply:
Yeah I was a bit torn on this. Everywhere else I use this (conftest JSON output, Gatekeeper logs) is JSON which uses `camelCase` and I'd like the key (variable here) to be consistent throughout.